core/common/src/main/java/alluxio/underfs/ObjectUnderFileSystem.java

```diff
@@ -538,7 +542,7 @@ public boolean mkdirs(String path, MkdirsOptions options) throws IOException {
 
   @Override
   public InputStream open(String path, OpenOptions options) throws IOException {
-    IOException thrownException = null;
+    IOException thrownException = new IOException();
```

Comment:
if we change retries as we're considering in https://alluxio.atlassian.net/browse/ALLUXIO-3111, we'll no longer need to deal with this class of null warnings

Reply:
Done